.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,67 @@
+name: Node.js integration
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  integration:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, macos-14, ubuntu-latest]
```

Comment:
macos-latest and macos-14 are exactly the same:
https://github.com/nodejs/gyp-next/actions/runs/9015778952/job/24771097889?pr=255
https://github.com/nodejs/gyp-next/actions/runs/9015778952/job/24771097430?pr=255

See "Set up job" -> "Operating System" and "Runner Image"

Reply:
Ah, thanks for pointing this out. I copied this from https://github.com/nodejs/gyp-next/blob/main/.github/workflows/node-gyp.yml#L13. I'll remove this duplication.